modules/indexer/code/bleve.go

```diff
@@ -116,10 +128,20 @@ func addUpdate(update fileUpdate, repo *models.Repository, batch rupture.Flushin
 		return nil
 	}
 
+	language, ok := enry.GetLanguageByExtension(update.Filename)
```

Comment:
If possible, move this language processing to a separate function so in the future it's easy to abstract for other indexer engines.

Reply:
Moved code language detection to separate module